cypress.config.js

```diff
@@ -4,16 +4,16 @@ import { clear } from './dataBase';
 
 module.exports = defineConfig({
   e2e: {
-    baseUrl: 'http://localhost:3000',
+    baseUrl: 'https://conduit.mate.academy',
     setupNodeEvents(on, config) {
       on('task', {
         generateUser() {
-          let randomNumber = Math.ceil(Math.random(1000) * 1000);
-          let userName = faker.name.firstName() + `${randomNumber}`;
+          let randomNumber = Math.random().toString().slice(4);
```

Comment:
The `randomNumber` generation using `Math.random()` is a good approach to ensure unique usernames.